packages/cx-flowcontainer/doc/FlowContainer.md

```diff
@@ -24,11 +34,11 @@ const Default = asFluidToken(
       ComponentWrapper: cxSpacing.Gutter,
     },
   },
-  WithComponentsVariations, // a token that would define list of components available for this flowcontainer
+  WithComponentsVariations, // a token that would define list of components available for this FlowContainer
```

Comment:
This otken does not exist.  Also we have not documented all available tokens - see [here](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/cx-flowcontainer/src/components/FlowContainer/tokens/cxFlowContainer.ts#L158-L168)

Reply:
@misterjones  let me know if need assistance here...

Reply:
We discussed removing the token descriptions, and allowing them to be documented in the API documentation, correct?